docs/examples/droplet_fusion/droplet_fusion.rst

```diff
@@ -0,0 +1,200 @@
+    scan.plot(channel='green',frame=framenr)
+    plt.show()
+
+.. image:: frame6.png
+
+If the droplets are in focus, the size of the droplet can be estimated from the 2D scan. 
+First, we use image segmentation to identify the two droplets in the image::
+
+    import math
+    from skimage.measure import label, regionprops
+
+    framenr = 2  # Choose a frame before the fusion event on which you want to identify and measure droplets
+
+    image = scan.get_image(channel='red')[framenr]
+    image = image / np.max(image)
+    threshold = 0.5
+    blobs = image > threshold
+    label_img = label(blobs)
+
+    plt.figure()
+    plt.subplot(2, 1, 1)
+    plt.title("original, normalized image")
+    plt.imshow(image)
+    plt.subplot(2, 1, 2)
+    plt.title(f"Identified objects")
+    plt.imshow(label_img)
+    plt.tight_layout()
+    plt.show()
+
+.. image:: image_segmentation.png
+
+Estimate the size of the droplet by measuring the length of the longest axis and the shortest axis and taking the average::
+
+    regions = regionprops(label_img)
+
+    fig, ax = plt.subplots()
+    ax.imshow(image, cmap=plt.cm.gray)
+    average_droplet_radii = np.array([])  # List of arrays to collect all droplet radii
+
+    for n, props in enumerate(regions):
+        y0, x0 = props.centroid
+        orientation = props.orientation
+        x1 = x0 + math.cos(orientation) * 0.5 * props.axis_minor_length
+        y1 = y0 - math.sin(orientation) * 0.5 * props.axis_minor_length
+        x2 = x0 - math.sin(orientation) * 0.5 * props.axis_major_length
+        y2 = y0 - math.cos(orientation) * 0.5 * props.axis_major_length
+
+        ax.plot((x0, x1), (y0, y1), "-r", linewidth=2.5)
+        ax.plot((x0, x2), (y0, y2), "-r", linewidth=2.5)
+        ax.plot(x0, y0, ".g", markersize=15)
+
+        minr, minc, maxr, maxc = props.bbox
+        bx = (minc, maxc, maxc, minc, minc)
+        by = (minr, minr, maxr, maxr, minr)
+        ax.plot(bx, by, "-b", linewidth=2.5)
+       
+        average_droplet_radii = np.append(
+            average_droplet_radii, 0.25 * (props.minor_axis_length + props.major_axis_length)
+        )
+    plt.title("The short and long axis of the identified objects")
+    plt.show()
+
+.. image:: droplet_size_pixels.png
+
+The array `average_droplet_radii` contains the average of the large and small radius for all the droplets in the image, in pixels. 
+Below we are multiplying this array by the pixel size in micron to obtain the radii in micron::
+
+    average_droplet_radii_um = average_droplet_radii * scan.pixelsize_um[0]
+
+The average radii for the droplets in micrometers are::
+
+    >>> print(average_droplet_radii)
+
+    [11.05117733 10.73977514]
+
+We now determined the relaxation time as well as the droplet sizes. The next step is to measure these two quantities for many different fusion events and plotting 𝜏 vs average radius.
+The slope of this plot is given by 𝜂/𝛾 (viscosity/surface tension) for a Newtonian fluid [2]_.
```

Comment:
Preference thing, but:
```suggestion
For a Newtonian fluid, the slope of this plot is given by 𝜂/𝛾 (viscosity/surface tension) [2]_.
```

Is this dynamic or kinematic viscosity? Might be nice to add units here.